src/derived/slot.rs

```diff
@@ -671,19 +690,24 @@ where
 
                     return None;
                 }
-
-                false
             }
         };
 
+        Some(self.mark_value_as_verified(revision_now))
+    }
+
+    fn mark_value_as_verified(&mut self, revision_now: Revision) -> StampedValue<Q::Value> {
+        let value = match &self.value {
+            Some(v) => v.clone(),
+            None => panic!("invoked `verify_value` without a value!"),
+        };
```

Comment:
Alternative

```rust
let value = self.value.clone().unwrap_or_else(|| panic!("invoked `verify_value` without a value!"))
```

Reply:
I sort of find the explicit match more readable, but it's also (very mildly) more efficient, since we only check if the value is `Some` once. I suppose post optimization that's perhaps not even true. =)

Reply:
In any case I think I'll leave as is...